addons/arsenal/Cfg3DEN.hpp

```diff
@@ -8,7 +8,7 @@ class Cfg3DEN {
                         class GVAR(DefaultLoadoutsListAttribute) {
                             property = QGVAR(DefaultLoadoutsListAttribute);
                             value = 0;
-                            expression = "if !(is3DEN) then {ace_arsenal_defaultLoadoutsList = _value};";
```

Comment:
@alganthe Was there a reason this was like this? Could I break something by using QUOTE?

Reply:
```cpp
expression = QUOTE(if !(is3DEN) then {GVAR(defaultLoadoutsList) = _value});
```
works, and so does
```cpp
expression = if !(is3DEN) then {GVAR(defaultLoadoutsList) = _value};
```

Reply:
I'll keep the verbose one with `QUOTE`, but good point. Might abuse it later.

Reply:
Second one just gets auto-quoted, and will throw missing quotations warning with armake (unless that warning is disabled).